woocommerce/woocommerce-ios
# [Woo POS] Update Checkout and Add More buttons in POS

## Description


- Updated Checkout and Add More buttons in POS
- Using HStack for button label so that the entire content of the button is tappable
- Disabling Checkout button if the cart is empty
- Updated button padding

## Testing information


- Enter POS experience
- With empty cart check that the Checkout button is disabled
- Add various items to cart, check that Checkout button is enabled
- Tap the Checkout button, check that tapping any part of the Checkout button works

## Screenshots


![Simulator Screen Recording - iPad Pro (11-inch) (4th generation) - 2024-06-03 at 15 50 02](https://github.com/woocommerce/woocommerce-ios/assets/6242034/c2015fe0-2204-403f-9087-a9e07abdd79b)

---
